lib/mastalk/snippets/tool.html.erb

```diff
@@ -0,0 +1,11 @@
+# $~cost-calc, ~$
+
+<div class="l-container-tool">
+  <iframe id="calculator_embed"
+		  frameborder="0"
+		  width="100%"
+		  class="calculator-preview__iframe"
+		  src="https://www.moneyadviceservice.org.uk/en/cost-calculator-builder/embed/calculators/<%= Mastalk::Document.new(body.strip).to_html.gsub(/<br \/>|<p>|<\/p>|\n/, '') %>">
```

Comment:
Just realised I think we need a default height of `1000` on this for non-js users